src/buckets/components/csvUploader/CsvUploaderWizard.tsx

```diff
@@ -68,11 +78,15 @@ const CsvUploaderWizard = () => {
         <OverlayFooter>
           <Button
             color={ComponentColor.Default}
-            text={uploadState === RemoteDataState.Loading ? 'Cancel' : 'Close'}
+            text={buttonText}
             size={ComponentSize.Medium}
             type={ButtonType.Button}
-            onClick={handleDismiss}
-            testID="lp-close--button"
+            onClick={
+              uploadState === RemoteDataState.Error
+                ? resetUploadState
+                : handleDismiss
+            }
+            testID="csv-close--button"
```

Comment:
made these changes to allow a user to clear out their current upload state when an error appears, rather than having the user have to close and open the modal / navigate to and away from the page

Reply:
Nice!  That sounds like much better UX to me :100: